src/infrastructure/figma/figma-service.ts

```diff
@@ -96,7 +93,7 @@ export class FigmaService {
 		issueKey: string;
 		issueTitle: string;
 		atlassianUserId: string;
-	}): Promise<CreateDevResourcesResponse> => {
+	}): Promise<void> => {
```

Comment:
**note:** We don't want to expose Figma API model from the service. The return type can be safely replaces with `void` since it is not used.